wasm/parsers/tables.py

```diff
@@ -49,3 +57,39 @@ def parse_table(stream: IO[bytes]) -> Table:
     """
     table_type = parse_table_type(stream)
     return Table(table_type)
+
+
+def parse_table_instruction(opcode: BinaryOpcode, stream: IO[bytes]) -> Instruction:
+    """
+    https://webassembly.github.io/spec/core/binary/instructions.html#table-instructions
+    """
+
+    if opcode is BinaryOpcode.TABLE_GET:
+        tableidx = parse_table_idx(stream)
+        return TableGet(tableidx)
+    if opcode is BinaryOpcode.TABLE_SET:
+        tableidx = parse_table_idx(stream)
+        return TableSet(tableidx)
+    if opcode is BinaryOpcode.TABLE_INIT:
+        elemidx = parse_elem_idx(stream)
```

Comment:
Why is the elem first and the table second?

Reply:
That is how it is defined in the [spec](https://webassembly.github.io/spec/core/binary/instructions.html#table-instructions). I don't know why the parameters are flipped in the binary format. 

[Text format](https://webassembly.github.io/spec/core/text/instructions.html#table-instructions) (`=>` denotes t2a conversion):

```
plaininstr ::= 'table.init' x:tableIdx y:elemIdx => table.init x y
```

vs [Binary format](https://webassembly.github.io/spec/core/binary/instructions.html#table-instructions):

```
instr ::= 0xFC 12:u32 y:elemidx x:tableidx => table.init x y
```